### ISSUE TITLE
TypeError: the JSON object must be str, bytes or bytearray, not dict

### ISSUE THREAD

**Issue by sadboys2001** [Kilvoctu/aiyabot#30]
![image](https://user-images.githubusercontent.com/14942848/199139264-860a197d-0076-4b76-9b93-10c5119b5b3a.png)
![image](https://user-images.githubusercontent.com/14942848/199139324-48acf808-0f84-4fbb-8854-0ebcdba65d5e.png)

I keep getting this error after the newest update and not sure what I'm doing wrong.

**Comment by Kilvoctu** [comment]
It looks to me like it's having trouble reading the info for the metadata/png info.

Did you update your Web UI repo?
What happened is at some point they updated the API response to make getting the metadata much more complicated. So when I updated, I had to change my code to retrieve the information the new way.

AIYA right now is tested on commit `198a1ffcfc963a3d74674fad560e87dbebf7949f` of the web ui.
